cfpb/cfgov-refresh
# WIP: Prepaid agreements search

Adds ability to search and download prepaid agreement files.

## Additions

- Prepaid agreements app

## Testing

1. Check out branch, run migrations and gulp, and load agreements data
   ```
   python cfgov/manage.py migrate
   python cfgov/manage.py runscript import_prepaid_agreements_data
   yarn run gulp
   ```
2. Turn on `PREPAID_AGREEMENTS_SEARCH` flag in admin
3. Navigate to [localhost:8000/prepaid-agreements](http://localhost:8000/prepaid-agreements) 


## Screenshots




## Notes
This is an initial PR for work that is still in progress. Some outstanding issues:

- We are still refining the relationship between the filters and search form. Currently the filters are narrowed to only display values represented in the results when a search term is submitted, but not when a filter is applied.

- Submitting the search and filter forms currently involves a page reload. We intend to add Javascript form submission based on the existing eRegs search page code in a future PR.

- Tests are in progress, particularly for the search/filter functionality that is still being finalized, but we would still appreciate feedback in the meantime


## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
